core/src/main/scala/cats/data/EitherT.scala

```diff
@@ -1019,7 +1019,7 @@ abstract private[data] class EitherTInstances extends EitherTInstances1 {
       type F[x] = Nested[P.F, Validated[E, *], x]
 
       implicit val monadM: Monad[M] = P.monad
-      implicit val monadEither: Monad[Either[E, *]] = cats.instances.either.catsStdInstancesForEither
+      // implicit val monadEither: Monad[Either[E, *]] = cats.instances.either.catsStdInstancesForEither
```

Comment:
Hum, what's going on here? Can we just remove it completely?

Reply:
fixed